.gitignore

```diff
@@ -1,3 +1,2 @@
-build
```

Comment:
Just so I understand, why did you remove this? The `build` dir should not be checked in.

Reply:
This was necessary in order to run the api against a dev branch of the catalog (included by pointing to dev branch in the github repo). Since this requires an install and compile it also needs the build files included in node_modules. Please let me know if there is way of doing this. Maybe we should remove the build directory again before merging into master.

Reply:
Removed in ae5a5492af009ff0687ccc0645c714d1caff512c.

Reply:
I explained local development in 249d4d12b2efcd9291f64cba337d66a910ceb41b. Please let me know if that clears up things.